ext/scripts/tests/install_scripts/assert.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+assert_test_failed=0
+
+function assert() {
```

Comment:
ok, I take it all back: Perl is better than bash


(please also ignore this comment)